hoot-core/src/main/cpp/hoot/core/io/ApiDbReader.cpp

```diff
@@ -292,12 +304,24 @@ void ApiDbReader::_readByBounds(shared_ptr<OsmMap> map, const Envelope& bounds)
         _getDatabase()->selectElementsByElementIdList(relationIds, TableType::Relation);
       while (relationItr->next())
       {
-        RelationPtr element = _resultToRelation(*relationItr, *map);
-        LOG_VART(element->toString());
-        map->addElement(element);
+        RelationPtr relation = _resultToRelation(*relationItr, *map);
+        map->addElement(relation);
+        LOG_VART(relation);
+        boundedRelationCount++;
       }
     }
   }
+
+  LOG_INFO(
```

Comment:
I did initially want this one at info, but I think it should be debug.  I'll change it back.